plugins/com.google.cloud.tools.eclipse.appengine.whitelist/src/com/google/cloud/tools/eclipse/appengine/whitelist/AppEngineJreWhitelist.java

```diff
@@ -1910,24 +1910,21 @@
               "javax.naming.ldap.Rdn"
               ));
   
-  // TODO: com.sun and com.oracle packages are tricky. Some are in the JRE. Some aren't. 
   
   /**
    * @param className fully package qualified class name
    * @return true if this class is allowed in Java 7 on App Engine Standard, false otherwise
    */
   public static boolean contains(String className) {
     if (className.startsWith("javax.") ){
-      if (isBundledInJre(className)) {
-        return WHITELIST.contains(className);
-      }
-      return true;
+      return !isBundledInJre(className) || WHITELIST.contains(className);
     } else if (className.startsWith("java.") 
```

Comment:
I had meant that all of these packages should be moved into isBundledInJre() too.